gostream/media.go

```diff
@@ -313,11 +313,14 @@ func (pc *producerConsumer[T, U]) start() {
 					}
 				}, err}
 				pc.currentMu.Unlock()
-				if lastRelease != nil {
```

Comment:
`lastRelease` "last" is ambiguously similar to "final" and confused me a bit on first glance